hardcodes.R

```diff
@@ -101,8 +100,8 @@ project_types_enrolled_homeless <- c(lh_project_types, 14)
    
 long_stayer_percentile_project_types <- c(0, 2, 3, 8, 9, 10, 12, 13)
 
-all_project_types <- c(0, 1, 2, 3, 4, 6, 8, 9, 10, 11, 12, 13, 14) # minus Other
-all_project_types2 <- c(0, 1, 2, 3, 4, 6, 7, 8, 9, 10, 11, 12, 14)
+all_project_types <- c(0, 1, 2, 3, 4, 6, 8, 9, 10, 11, 12, 13, 14) 
+# minus Other^
```

Comment:
Just wanted to confirm that though we are excluding Other in this vector, that doesn't have an impact on Inflow/Outflow logic where we do reference Other.

In what scenarios do we use this vector as opposed to a vector that _does_ include Other along with all other project types?

Reply:
We only ever use this in the hardcodes.R script to build the grid of what elements are required to be answered by whom (see the inc_ncb_hi_required_prep data frame at the bottom.)
If we truly want all project types, we wouldn't need to filter by ProjectType at all, and in that case Other would come in with everything else. I used "all" here to mean all Project Types defined by HUD. (I'll add a comment to that effect.) Let me know if you think we should be more explicit in the vector name.

Reply:
I think as long as the comment details why Other is excluded, the vector name is fine as-is.